### DISCUSSION TITLE
FireFly Data Importer request for auth everytime

### DISCUSSION THREAD

**Discussion by fedex03** [firefly-iii/firefly-iii#7097] [Firefly III]
Hi there,

I'm using FireFlyIII on docker and I really love it!
I also use FireFly Data Impoter on docker, but I have a strange behaviour.
Everytime I access the main page of FireFly Data Impoter, it ask my to do the authentication procedure.

How can I avoid this?

Thank you!

**Comment by JC5** [comment]
What is your configuration like?

**Reply by fedex03** [reply]
Firefly IP: http://192.168.10.2:8082
FIDI: http://192.168.10.2:8081

**Reply by JC5** [reply]
And your authentication config?

**Reply by fedex03** [reply]
This is the Callback URL: http://192.168.10.2:8081/callback

I added the client ID and it works very well but everytimes open the Import Page again I need to authenticate again.

**Reply by JC5** [reply]
Yes, that's the point of setting the client ID. So that's correct behavior.

I expect you'll want to set an access token instead.

**Reply by fedex03** [reply]
I go to: Profile-->Client OAuth-->Add New Client.
Here I past the Callback URL and I remove the Reserved flag.
Now I get the ID in this case is 5 and paste it into the FileImporter togheder with the FireFly URL.

Is this the right procedure?

Thnak you very much for your time!

**Reply by JC5** [reply]
At the bottom, create a Personal Access Token.

https://docs.firefly-iii.org/data-importer/install/configure/#configure-fidi

**Reply by fedex03** [reply]
I created the Personal Access Token. 
What should I do with this token?

Thnak you

**Reply by JC5** [reply]
https://docs.firefly-iii.org/data-importer/install/configure/#configure-fidi

**Reply by fedex03** [reply]
I forget to mention my File Importer is version "Firefly III Data Import Tool, v1.0.0"
